evgMrmApp/Db/evgDbus.db

```diff
@@ -34,6 +34,7 @@ record(mbbo, "$(P)Src-Sel") {
 }
 
 record(mbbo, "$(P)MapConv-Sel_") {
+    field(ASG, "private")
```

Comment:
Maybe keep the same formatting as the other fields ? :
`    field( ASG, "private")`

Reply:
Good point, but some files follow this formatting and others are different.